BackendWebApp/src/main/java/uk/ac/ebi/biostd/webapp/application/validation/eutoxrisk/dto/EUToxRiskFileValidationError.java

```diff
@@ -0,0 +1,43 @@
+package uk.ac.ebi.biostd.webapp.application.validation.eutoxrisk.dto;
+
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+/**
+ * @author Olga Melnichuk
+ */
+@Getter
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class EUToxRiskFileValidationError {
+
+    private String sheet;
+    private int line;
+    private int column;
+    private String message;
+
+    public EUToxRiskFileValidationError() {
+    }
+
+    private EUToxRiskFileValidationError(String sheet, int line, int column, String message) {
+        this.sheet = sheet;
+        this.line = line;
+        this.column = column;
+        this.message = message;
+    }
+
+    public static EUToxRiskFileValidationError serverError(String msg) {
+        return new EUToxRiskFileValidationError("unknown", 0, 0, msg);
+    }
+
+    @Override
+    public String toString() {
```

Comment:
you can use lombok `@ToString` which produce a great result, please take a look

Reply:
Good to know! I just have this concern that we'll have to change it anyway in the future, as it should be visible to the end user in the SubmTool..

Reply:
all done: added `@ToString` and removed ovierriden method